profile/2018-06-01-profile.json

```diff
@@ -0,0 +1,656 @@
+{
+    "info":{
+
+        "name":"2018-06-01-profile",
+
+        "description":"Profile definition targeted for applications that could run on public Azure and the Sovereign clouds."
+    },
+
+    "resource-manager": {
+
+        "microsoft.analysisServices": {
+                        
+                        "2017-08-01": [
+                            "locations",
+                            "locations/checkNameAvailability",
+                            "locations/operationresults",
+                            "locations/operationstatuses",
+                            "operations",
+                            "servers"
+                        ]
+        },
+
+        "microsoft.authorization": {
+
+                        "2015-06-01": [
+                            "classicAdministrators"
+                        ],
+                        
+                        "2017-04-01": [
+                            "locks"
+                        ],
+
+                        "2017-05-01":[
+                            "elevateAccess",
+                            "operations",
+                            "permissions",
+                            "providerOperations",
+                            "roleDefinitions"
+                        ],
+                       
+                        "2017-09-01":[
+                            "checkAccess",
+                            "roleAssignments"
+                        ],
+
+                        "2018-03-01":[
+                            "policyAssignments",
+                            "policyDefinitions",
+                            "policySetDefinitions"
+                        ]
+            }
+
+        },
+
+        "microsoft.Batch": {
+
+                         "2017-05-01": [
+                             "batchAccounts",
+                             "locations",
+                             "locations/quotas",
+                             "operations"
+                         ]
+        },
+
+        "microsoft.cache": {
+
+                         "2017-10-01":[
+                             "redis",
+                             "checkNameAvailability",
+                             "locations",
+                             "locations/operationResults",
+                             "operations"
+                         ]
+
+        },
+
+        "microsoft.classicCompute": {
```

Comment:
Without Swagger definitions, these RPs don't mean anything to us. If this is an attempt to provide multiple Compute versions into a profile, it seems like there are some anti-patterns trying to make their way in.

Reply:
What I was trying to say.

Reply:
To be more constructive about this, the way we would recommend doing this would be to have two profiles.

Reply:
@sphibbs FYI

Reply:
@marstr @dsgouda Microsoft.ClassicCompute is RDFE compat, it's another RP:
https://docs.microsoft.com/en-us/azure/role-based-access-control/resource-provider-operations#microsoftclassiccompute

https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-move-resources#new-subscription